examples/sslstrip.py

```diff
@@ -0,0 +1,41 @@
+from netlib.http import decoded
+import re
+from six.moves import urllib
+
+def start(context, argv) :
+
+    #set of SSL/TLS capable hosts
+    context.secure_hosts = set()
+
+def request(context, flow) :
+
+    flow.request.headers.pop('Accept-Encoding', None)
```

Comment:
Now that you mention it, I don't think stripping that is necessary. It's part of Moxie's script but probably just because his proxy can't handle gzipped responses. mitmproxy should be just fine.